File: slo.ml
#!/usr/bin/env ocaml
type point = {x: int; y: int};;

let total = ref 0 in
for i = 1 to 1000000 do
  let p = {x=2; y=3} in
  total := !total + p.x + p.y
done;

print_int !total;
print_newline ()

File: slo.js
#!/usr/bin/env node
class Point {
  constructor(x, y) {
    this.x = x;
    this.y = y;
  }
}

let total = 0;
for (let i = 0; i < 1000000; i++) {
  const p = new Point(2, 3);
  total = total + p.x + p.y;
}
console.log(total);
